src/components/Donation/machines/donationType.ts

```diff
@@ -76,13 +76,13 @@ export type DonationMachineContext = {
    * A key to hold the response once the donation service respond
    */
   donation: {
-    message: string;
+    message?: string;
     status: string;
+    /**
```

Comment:
Looking at the code seems that the membership machine has better type definition than the donation one. Once this Pull request get merged we can unify both widgets https://app.asana.com/0/1195530778765030/1199537989678029